packages/winston-log-enricher/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## 3.1.0 (05/05/2022)
+
+ * Added a log line to log aggregator regardless of transaction context when application logging forwarding is enabled.
+ * Added NR-LINKING metadata when local log decorating is enabled.
+ * Cleaned up package to appropriately handle all logging use cases.(log enriching, log forwarding, local log decorating, and supportability metrics on load)
```

Comment:
instead of having these in parens, maybe having them as sub-bullets since it's easier to read?